client/src/components/Shared/Form/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import img from './cat.jpeg';
+
+import {
+  StyledForm,
+  StyledImg,
+  StyledInformation,
+  StyledNotic,
+  StyledDiv,
+  StyledLabel,
+  StyledSelect,
+  StyledOption,
+  StyledItem,
+  StyledLabels,
+  StyledInput,
+  StyledTextarea,
+  StyledImgCon,
+}
+  from './form.style';
+
+const Form = () => (
+  <StyledForm>
+    <StyledImgCon>
+      <StyledImg src={img} />
+    </StyledImgCon>
+
+    <StyledInformation>PIECE INFORMATION</StyledInformation>
+    <StyledNotic>Please verify the items marked * </StyledNotic>
+
+
+    <StyledDiv>
+
+      <StyledLabels>
+
+        <StyledLabel> Item Type * </StyledLabel>
```

Comment:
There doesn't need to be a space for the *s.

Reply:
![Screenshot from 2019-03-24 11-47-19](https://user-images.githubusercontent.com/32957970/54877710-9d876580-4e2a-11e9-9db0-637fc7a4bf76.png)
it's look like this without space